intelsdi-x/snap-plugin-collector-mesos
# fixed examples, vagrant file and bumped version to make plugin loadab…

Summary of changes:
- Fixed examples to use proper plugin name
- Changed build system to glide
- Updated dependencies to use Snap's 1.0 libraries
- Bumped plugin version and tagged it appropriately to release plugin  loadable with Snap 1.0
- Updated Vagrantfile and related script to download plugins for Snap 1.0

How to verify it:
- Run examples
- Run `vagrant up` and check if everything behaves correctly

Testing done:
- Manual tests
